LukeMason33/refactor-tractor
# Fix/unit testing

## Pull Request

<br>

### Description

* Is this a feature or a fix?
This is a fix. It contains the complete redo of our unit tests for our two classes, User and Recipe. All methods and properties are tested using sample data sets that are located in the data directory. These data miles mock the data that is returned from our network requests in the fetch.js file.

* Where should the reviewer start?
The reviewer should start in the test directory.

* What functionalities do these changes effect? What was the motivation behind these changes?
This does not change any of the functionality within our site. But there was a last minute commit on getting the cost of a recipe to display in the recipe cards.

* What outside features or research did you use to implement this fix?
I was using Postman to see how the data was being returned and wrote the tests according to that.

* How should this be tested?
This should be tested by running "npm start" in your terminal to ensure all the unit testing is passing.

* Applicable screenshot(s): 

<br>

***

#### Please review considerations below:
